docs/source/dev_guide/release.rst

```diff
@@ -68,6 +68,14 @@ Releasing on GitHub: release candidates
 
 1. Create a tag for the release candidate at https://github.com/E3SM-Project/zppy/tags.
 
+     ::
+
+	$ git checkout main
+	$ git fetch upstream
+	$ git rebase upstream/main
```

Comment:
```
$ git reset --hard upstream/main
```
This would be better because you don't typically want to keep any changes you might have made in your local `main`.

Reply:
Ah, I typically don't make changes on my local `main`, but I see how this command would be more fail-safe. Thanks!

Reply:
Added as part of #440.